mpd/mpd.go

```diff
@@ -159,14 +194,15 @@ type SegmentTemplate struct {
 }
 
 type Representation struct {
+	ID *string `xml:"id,attr"` // Audio + Video
```

Comment:
It's more normal to have `CommonAttributesAndElements` as the first line in the struct - this can go back to where it was

Reply:
Normal I'd agree but in this case, the inner CommonAttributesAndElements would get picked up first by XMLMarshal and it's ordering by struct order. (no guarantee this will always be the case but..) XML doesn't care about the order of attr , but it's more human readable to have the ID at the front IMO.